Documentation/gettingstarted/cli-download.rst

```diff
@@ -16,6 +18,8 @@ and enable/disable a variety of functionality.
     .. code-block:: shell-session
 
       curl -LO https://github.com/cilium/cilium-cli/releases/latest/download/cilium-darwin-amd64.tar.gz
+      curl -LO https://github.com/cilium/cilium-cli/releases/latest/download/cilium-darwin-amd64.tar.gz.sha256sum
+      sha256sum --check cilium-darwin-amd64.tar.gz.sha256sum
```

Comment:
The `sha256sum` binary does not exist on MacOS. Use `shasum -a 256` instead.
```suggestion
      shasum -a 256 -c cilium-darwin-amd64.tar.gz.sha256sum
```

Reply:
Oops 🙈